### DISCUSSION TITLE
Darkmode Contrast Unreadable

### DISCUSSION THREAD

**Discussion by emarshall87** [laravel/nova-issues#5348] [Ideas & Feature Requests]
- Laravel Version: 9.52.0
- Nova Version: 4.22.0
- PHP Version: 8.1
- Database Driver & Version:
- Operating System and Version: MacOS Ventura 13.2.1
- Browser type and version: Chromium

### Description:
The text in the dark mode version of Nova has very little contrast, making it very difficult to use. While I usually prefer dark mode, I am forced to use light mode with Nova 4. Ideally, the text colors would be made white or near-white to give appropriate color contrast against the dark background.

Additionally, I do not see any documentation for modifying the text-color values for dark mode so am not able to easily fix this.

### Detailed steps to reproduce the issue on a fresh Nova installation:
This happens on any out of the box configuration with Nova 4 where the user's system settings are dark mode (so Nova defaults to dark mode).

**Comment by vedmant** [comment]
Also for me dark more has bad contract, text should be lighter, currently it's hard to read, and there is no option to fix this, unless to search replace all classes in the sources from like `dark:text-gray-400` to `dark:text-gray-200` and other similar.
